consensus/polybft/consensus_runtime.go

```diff
@@ -991,12 +991,34 @@ func (c *consensusRuntime) IsValidBlock(proposal []byte) bool {
 }
 
 func (c *consensusRuntime) IsValidSender(msg *proto.Message) bool {
```

Comment:
We can discuss whether sender validation belongs to the `consensusRuntime` or the `FSM`. This suggestion came from @goran-ethernal and I agree, because `consensusRuntime` already has enough knowledge to validate sender on its end. No need to have indirection to the `FSM`